packages/material-ui/src/Slider/Slider.test.js

```diff
@@ -36,6 +36,10 @@ describe('<Slider />', () => {
     inheritComponent: SliderUnstyled,
     mount,
     refInstanceof: window.HTMLSpanElement,
+    muiName: 'MuiSlider',
+    ThemeProvider,
+    createMuiTheme,
```

Comment:
ThemeProvider and createMuiTheme seems to be invariant. Why provide them in each test?

Reply:
I cannot add a dependency to them in the `test/utils` right? How would I use them. Or am I missing something?

Reply:
> I cannot add a dependency to them in the test/utils right?

Why not? I was wondering about importing the theme directly in `test/utils`.

Reply:
Nevermind fixed, I thought I would create circular dependencies if I add imports to the core, but it works just fine